src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/runner/DeletedAccountsApiTest.java

```diff
@@ -80,5 +87,91 @@ public void assertHttpForbiddenWithInvalidS2SToken() throws JsonProcessingExcept
         );
     }
 
+    @Test
+    @Title("Assert response code of 200 for successfully getting deleted records")
+    public void assertHttpSuccessResponseCodeForGetRequestCaseViewApi() throws JSONException {
```

Comment:
Name doesn't make sense here, it is not CaseViewApi, it is DeletedAccountsApi (I see that similar wrong name is also used for Post request, my bad)